### ISSUE TITLE
booleans must be in lower case

### ISSUE THREAD

**Issue by datarishi** [mitya33/cfimp#4]
When passing a .csv file with boolean values in upper case, i.e. TRUE,FALSE the following error message was produced:

`12:06:23 - InvalidEntry: Status: 422 - Unprocessable Entity - Message: Validation error - Details: The type of "value" is incorrect, expected type: Boolean, The type of "value" is incorrect, expected type: Boolean, The type of "value" is incorrect, expected type: Boolean - Request ID: 79dd5de1-5363-4f36-a474-9c2abe4bb313`

I was encountering this frequently as Excel defaults to capitalising booleans if cell format is set to General rather than Text.

So not necessarily an issues, as the workaround is trivial, but perhaps an easy fix or something to mention in documentation.

**Comment by mitya33** [comment]
Yeah, as you've noticed cfimp auto-parses true/false strings as booleans only if they're lower-case. I'll amend it to be case-insensitive but close this off for now since there's an obvious work-around. Thanks.
